stock-plugins/aiverify.stock.accumulated-local-effect/algorithms/accumulated_local_effect/plugin_init.py

```diff
@@ -0,0 +1,86 @@
+import sys
+import argparse
+from test_engine_core.plugins.enums.model_type import ModelType
+from accumulated_local_effect.algo_init import AlgoInit
+
+# Global variables
+data_path = ''
+model_path = ''
+ground_truth_path = ''
+ground_truth = ''
+run_pipeline : bool = True
+model_type = ''
+core_modules_path = ''
+
+def run():
+    parse_input_args()
+    print_user_input()
+    invoke_accumulated_local_effect_plugin()
+
+def parse_input_args():
+
+    global data_path, model_path, ground_truth, ground_truth_path, run_pipeline, model_type
+
+    parser = argparse.ArgumentParser(description='Process comma-separated arguments.')
+    
+    # Add a command-line argument that accepts a comma-separated string
+    parser.add_argument('input', type=str, help='Comma-separated values')
```

Comment:
Let's not parse manually and use the framework itself. It would look something like this:

```py
    parser = argparse.ArgumentParser(
        description="Run the plugin test with specified parameters."
    )

    parser.add_argument("--data_path", required=True, help="Path to the data file.")
    parser.add_argument("--model_path", required=True, help="Path to the model file.")
    parser.add_argument(
        "--ground_truth_path", required=True, help="Path to the ground truth data file."
    )
    parser.add_argument(
        "--ground_truth",
        required=True,
        help="The ground truth column name in the data.",
    )
    parser.add_argument(
        "--run_pipeline",
        action=argparse.BooleanOptionalAction,
        default=False,
        help="Whether to run the test as a pipeline (default: False).",
    )
    parser.add_argument(
        "--model_type",
        required=True,
        choices=["CLASSIFICATION", "REGRESSION"],
        help="The type of model (CLASSIFICATION or REGRESSION).",
    )
    parser.add_argument(
        "--core_modules_path",
        default="",
        help="Path to the core modules (default: empty).",
    )

    # Parse arguments
    args = parser.parse_args()

    # Debugging prints
    print(
        f"Running with the following arguments:\n"
        f"Data Path: {args.data_path}\n"
        f"Model Path: {args.model_path}\n"
        f"Ground Truth Path: {args.ground_truth_path}\n"
        f"Ground Truth: {args.ground_truth}\n"
        f"Run Pipeline: {args.run_pipeline}\n"
        f"Model Type: {args.model_type}\n"
        f"Core Modules Path: {args.core_modules_path}"
    )

    # Map string argument to ModelType enum
    model_type = ModelType[args.model_type]
```